### PR TITLE
Newsletter sign up

### PR DESCRIPTION
**Pull request by grantglidewell**
Adds newsletter signup form.
Adds lambda function to handle signup.

Also migrates functionality into Netlify that previously lived in Zeit now functions. This merge needs to coincide with those services being turned off. Please contact me directly to merge and deploy this change.

### CODE REVIEW

**Comment by grantglidewell** [comment]
Definitely squash merge this, as the number of commits is inflated due to how Netlify forms and lambdas behave (they must be tested via live/preview deploys).

File: functions/submission-created.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        url: `https://api.pipedrive.com/v1/notes?api_token=${PIPEDRIVE_KEY}`,
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        data: JSON.stringify({
+          content: website,
+          deal_id,
+        }),
+      });
+    } catch (err) {
+      console.error('error creating note', err);
```
Comment (daceej) [inline functions/submission-created.js]: Will these errors be logged somewhere such that we would know when there is a failure?

File: functions/submission-created.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      callback(null, { statusCode: 200 });
```
Comment (daceej) [inline functions/submission-created.js]: Just confirming that we want to report a success code on a failure?

File: functions/submission-created.js (hunk 1 above)
Comment (grantglidewell) [reply functions/submission-created.js]: Yeah, the lambda function in Netlify have a log

File: functions/submission-created.js (hunk 2 above)
Comment (grantglidewell) [reply functions/submission-created.js]: Yes, unfortunately the chain of events here is not reliably quick enough to handle errors on the front end. we have all the data in the netlify forms so if an error does occur we have the ability to piece it together between the logs, what is in our service, and what the form submission looks like in netlify.

**Pull request merged**